Content.Shared/_RMC14/Marines/Roles/Ranks/SharedRankSystem.cs

```diff
@@ -76,17 +76,18 @@ public void SetRank(EntityUid uid, ProtoId<RankPrototype> from)
     }
 
     /// <summary>
-    ///     Gets the rank name of a given mob.
+    /// Gets the rank name of a given mob.
```

Comment:
The rest of the file (and default by .editorconfig) is to have 4 spaces before the comment, you don't need to make this change
```suggestion
    ///     Gets the rank name of a given mob.
```